jsonate/fields.py

```diff
@@ -1,3 +1,4 @@
+from builtins import str
```

Comment:
should be `from past import basestring` and leave the rest alone. This code is checking whether it received json data, which in python 3 _should_ be bytes (and in python 2, just a str). This trick assumes you are dealing with true strings (of text, not encoded data)